### DISCUSSION TITLE
Server Side Characters

### DISCUSSION THREAD

**Discussion by ClassyCustom** [Pryaxis/TShock#1697] [Q&A] [answered]
When the server sided character is enabled it applies the default kit of 3 copper tools , 100hp and 20mana but when you leave and join again it resets the hp and mana back to default even if you have used mana or life crystals.

**Comment by bippity** [comment, marked as answer]
If you're a superadmin, you have access to all permissions (*), including `tshock.ignore.ssc`.
That permission allows you to bypass SSC's management which essentially lets you play with your actual player's file. The server won't save your character, every time you join it will just load up your singleplayer character's file (which in your case is a new character).

**Simply create another group with the permission `*` (since the superadmin group can't be altered) and also give it `!tshock.ignore.ssc` which negates that permission.**
Example:
`1) /group add owner`  -Creates a group called "owner"
`2) /group addperm owner *` - Grants the owner group all permissions
`3) /group addperm owner !tshock.ignore.ssc` - Negates the ssc bypass permission
`4) /user group <your username> owner` - Sets your user to the Owner group
`5) /group prefix owner "[Owner] "` - Optional; sets a prefix for the Owner group

Executing those commands mentioned above will set your user to the "Owner" group and grants you the same thing as Superadmin but still allows you to save your character into SSC.
